bindings/java/iota-wallet-java/examples/src/main/CallbackEvents.java

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import org.iota.Wallet;
+import org.iota.api.NativeApi;
+import org.iota.types.*;
+import org.iota.types.ClientConfig.ApiTimeout;
+import org.iota.types.account_methods.GenerateAddresses;
+import org.iota.types.account_methods.SyncAccount;
+import org.iota.types.events.Event;
+import org.iota.types.events.EventListener;
+import org.iota.types.events.transactionprogress.TransactionProgressEvent;
+import org.iota.types.events.wallet.WalletEventType;
+import org.iota.types.exceptions.WalletException;
+import org.iota.types.ids.account.AccountAlias;
+import org.iota.types.secret.StrongholdSecretManager;
+
+public class CallbackEvents {
+
+    static class Callback implements EventListener {
```

Comment:
Is this a default implementation for an EventListener or? Looks like this is not used, so is this needed here?

Reply:
Fixed